dotnet/samples/HomeAutomation/appsettings.Development.json

```diff
@@ -1,5 +0,0 @@
-{
-  "AzureOpenAI": {
```

Comment:
Is it intentional to delete instead of move this file?

Reply:
The file is ignored by git, this should have no problem as in the readme we suggest using User Secrets instead of appsettings configuration.